RELEASE.md

```diff
@@ -1,42 +1,37 @@
 # How to release Kuadrant Operator
 
-## Kuadrant Operator Release
-
-To release a version _“v0.W.Z”_ of Kuadrant Operator in GitHub and Quay.io, follow these steps:
-
-### Release file format.
-This example of the `release.yaml` file uses tag `v1.0.1` as reference.
-
-```yaml
-# FILE: ./release.yaml
-kuadrant:
-  release: "1.0.1"
-olm:
- channels:
- - "alpha"
- default_channel: "alpha"
-dependencies:
- Authorino: "0.16.0"
- Console_plugin: "0.0.14"
- DNS: "0.12.0"
- Limitador: "0.12.1"
- Wasm_shim: "0.8.1"
-```
-The `kuadrant` section relates to the release version of the kuadrant operator.
-While the `olm` section relates to fields required for building the olm catalogs.
-And the `dependencies` section relates to the released versions of the subcomponents that will be included in a release.
-There are validation steps during the `make prepare-release` that require the dependencies to be release before generating the release of the Kuadrant operator.
-
+To release a version _“vX.Y.Z”_ of Kuadrant Operator in GitHub and Quay.io, there are two options, a manual and an automated process.
+For both processes, first make sure every [Kuadrant Operator dependency](https://github.com/Kuadrant/kuadrant-operator/blob/main/RELEASE.md#kuadrant-operator-dependencies) has been already released.
+
+## Automated Workflow
+
+1. Run the GHA  [Automated Release](https://github.com/Kuadrant/kuadrant-operator/actions/workflows/automated-release.yaml)
+   filling the following fields:
+   - gitRef: Select the branch/tag/commit where you want to cut a release from.
+   - kuadrantOperatorVersion: the [Semantic Version](https://semver.org/) of the desired release.
+   - authorinoOperatorVersion: Authorino Operator bundle version (X.Y.Z)
+   - limitadorOperatorVersion: Limitador Operator bundle version (X.Y.Z)
+   - dnsOperatorVersion: DNS Operator bundle version (X.Y.Z)
+   - wasmShimVersion: WASM Shim version (X.Y.Z)
+   - consolePluginVersion: ConsolePlugin version (X.Y.Z)
+   - olmChannel: This will set the OLM `channels` and `default-channel` annotations
+2. The workflow will create a Pull Request that should be peer-reviewed and approved by a member of the Kuadrant team, focusing on the changes made in Kustomize config, OLM bundles and Helm Charts.
+3. Once the PR is merged, a release workflow will be triggered tagging and publishing the [Github release](https://github.com/Kuadrant/kuadrant-operator/releases)
+   it will also build the images and packages and publish them on Quay, Helm repository.
+
+### Notes
+* It's not possible to cherry-pick commits, the workflow will pick a branch/tag/commit and all the history behind to the PR.
```

Comment:
This related to this, would also affect the manual approach. 

The PR is opened with the changes from the `prepare-release`, and some thing is missing from the release, lets say missing commit from main. That missing commit can be cherry picked to the branch which is the base targeting the release branch.

What should happen in that case? I am assuming the history of the base branch should be reordered to have the cherry picked commit before the release commit. The same would go for if during the review a wrong version is noticed. Would that be adding in a new commit or the release commit edited?

Reply:
Basically any change in the history of the branch could be done in the PR, manually.

Reply:
That I understand. The question would be should the history be reordered to ensure the `Prepare release` commit is the head of the branch?